docs/development/process.md

```diff
@@ -37,6 +37,16 @@ v1.33   | Week 39-40  | September 27, 2021     | October 10, 2021   | [@voelzmo]
 Apart from the release of the next version, the release responsible is also taking care of potential hotfix releases of the last three minor versions.
 The release responsible is the main contact person for coordinating new feature PRs for the next minor versions or cherry-pick PRs for the last three minor versions.
 
+### Release of helm charts
+
+The helm charts are released in lockstep with gardener releases. Check [the discussion in #4123](https://github.com/gardener/gardener/issues/4123) for more information about this decision.
+
+Therefore, the helm chart versions for the [controlplane](../../charts/gardener/controlplane/Chart.yaml) and [gardenlet](../../charts/gardener/gardenlet/Chart.yaml) charts have to be bumped during the release process.
+
+Set the version to the same as the gardener version that is to be released, but remove the `v` in front of the version number for a valid semver version.
+
+Once the changes are commited/merged to master, the [Chart releaser action](../../.github/workflows/release-charts.yaml) will release the new version and publish it to the repository at https://gardener.github.io/gardener/.
+
```

Comment:
This should be automated via a `prepare_release` step in the CI/CD system. No human will do it manually. cc @ccwienk

Reply:
I agree! I think I can pull the version from the VERSION file, is that correct?

Reply:
I think you can get some inspiration from https://github.com/gardener/gardener/blob/e84d5ab32a96b722284e89252ce992615ec4f600/hack/.ci/prepare_release, namely https://github.com/gardener/gardener/blob/e84d5ab32a96b722284e89252ce992615ec4f600/hack/.ci/prepare_release#L71. This script is used in `gardener-extension-*` repositories to auto-generate the `example/controller-registration.yaml` files during a new release (e.g. https://github.com/gardener/gardener-extension-provider-aws/commit/6ae742f41b8c9d1bf03d7db2b234e002044ae702). Without such script, the only change during a "release commit" is bumping the `VERSION` file (e.g. https://github.com/gardener/gardener/commit/e75d09648f3a1d95da4aa5aca121e08c726a95e4).
In your case, the changes in the `Chart.yaml` should become part of such a "release commit".

Reply:
I tried to find out what triggers the „release commit“ in g/g, but I can’t find it. As it’s the bot, I suspected it was an action, but there’s no matching one.

Did I miss something? Can you point me to where I need to incorporate that change?

Reply:
> I tried to find out what triggers the „release commit“ in g/g,

This is implemented deep down in the https://github.com/gardener/cc-utils repository, @ccwienk or @AndreasBurger can explain better.

> Can you point me to where I need to incorporate that change?

The only thing you need to provide is such a `prepare_release` script which performs your changes. Once the PR is merged, a Gardener maintainer has to enhance the pipeline definitions to make the CI/CD system executing your `prepare_release` script when a new Gardener release is triggered.

Reply:
Do I understand it correctly that those scripts are run in an alpine docker container? If not, which environment do I have to assume?

Reply:
Yes, I think it's `alpine`. Not sure if this is configurable meanwhile.